hudi-common/src/main/java/org/apache/avro/io/NoWrappingJsonEncoder.java

```diff
@@ -0,0 +1,40 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+package org.apache.avro.io;
+
+import org.apache.avro.Schema;
+import org.apache.avro.io.parsing.Symbol;
+
+import java.io.IOException;
+import java.io.OutputStream;
+
+public class NoWrappingJsonEncoder extends JsonEncoder {
+  public NoWrappingJsonEncoder(Schema sc, OutputStream out) throws IOException {
+    super(sc, out);
```

Comment:
Can we give some document.

Reply:
Added. It's mainly needed to handle union types. Avro's JsonEncoder does not handle by default.  I used something similar to https://github.com/allegro/json-avro-converter/blob/master/converter/src/main/java/org/apache/avro/io/NoWrappingJsonEncoder.java but removed unused constructors and other code.

Reply:
Just give the author some credit in the doc should be fine.